src/jvm.rs

```diff
@@ -178,6 +180,48 @@ impl<'jvm> Jvm<'jvm> {
     }
 }
 
+pub struct JvmBuilder<'args> {
+    args_builder: InitArgsBuilder<'args>,
+}
+
+impl<'args> JvmBuilder<'args> {
+    fn new() -> Self {
+        let args_builder = InitArgsBuilder::new()
+            .version(jni::JNIVersion::V8)
+            .option("-Xcheck:jni");
+        JvmBuilder {
+            args_builder
+        }
+    }
+
+    pub fn add_classpath(mut self, classpath: impl Display) -> Self {
+        self.args_builder = self.args_builder.option(format!("-Djava.class.path={classpath}"));
+        self
+    }
+
+    pub fn custom(mut self, opt_string: impl AsRef<str> + Into<Cow<'args, str>>) -> Self {
+        self.args_builder = self.args_builder.option(opt_string);
+        self
+    }
+
+    fn build_jvm(self) -> crate::GlobalResult<JavaVM> {
+        let jvm_args = self.args_builder.build()?;
+        let jvm = JavaVM::new(jvm_args)?;
+        Ok(jvm)
+    }
+
+    /// Launch a new JVM, returning an error if one already exists.
+    pub fn launch<R>(
```

Comment:
I suspect most users would want to configure the JVM at the beginning but not actually "launch" it. Something like:

```rust
fn main() {
    Jvm::builder().foo.bar.build()?;
    ...
    do_stuff();
}

fn do_stuff() {
    Jvm::with(|jvm| { ... });
}
```

This could just be an alias for `launch(|_| Ok(()))`.

I *suspect* it would also be useful to have something `try_build` that doesn't consider it an error if the jvm is already launched. 

But we can do this in a later PR.